config/defaults.mk

```diff
@@ -374,7 +374,7 @@ UMPIRE_OPT = -I$(UMPIRE_DIR)/include
 UMPIRE_LIB = -L$(UMPIRE_DIR)/lib -lumpire
 
 # AMGX library configuration
-AMGX_DIR = @MFEM_DIR@/../amgx
+AMGX_DIR = @MFEM_DIR@/../AMGX
```

Comment:
@am-oler could you leave as amgx (lower case)? The CEED benchmarks build packages using lowercase names.